pyvista/__init__.py

```diff
@@ -74,3 +54,45 @@
 
 # Name used for unnamed scalars
 DEFAULT_SCALARS_NAME = 'Data'
+
+MAX_N_COLOR_BARS = 10
+
+
+# Lazily import/access the plotting module
+def __getattr__(name):
+    """Fetch an attribute ``name`` from ``globals()``or the ``pyvista.plotting`` module..
+
+    This override is implemented to prevent importing all of the plotting module
+    and GL-dependent VTK modules when importing PyVista.
+
+    Raises
+    ------
+    AttributeError
+        If the attribute is not found.
+
+    """
+    try:
+        return globals()[name]
+    except KeyError:
+        pass
+
+    import importlib
+    import inspect
+
+    whitelist = [
+        'demos',
+        'examples',
+        'ext',
+        'trame',
+        'utilities',
+    ]
+    if name in whitelist:
+        return importlib.import_module(f'pyvista.{name}')
+
+    _module = importlib.import_module('pyvista.plotting')
```

Comment:
I know we discussed something like this recently, but isn't this just `import pyvista.plotting as plotting` (instead of `_module` if we do this)?

Reply:
Yes, will change to `import ...` syntax